### ISSUE TITLE
CIRCE codeset 'Select concept...' should use a case insensitive search

### ISSUE THREAD

**Issue by pbr6cornell** [OHDSI/Circe#1]
Currently, 'Select concept...' uses a case sensitive search, so 'Acute myocardial infarction' returns a different set than 'acute myocardial infarction'.  In HERMES, I think we settled on making search case-insensitive.  I'd apply the same convention here (not sure if this issue should be moved to WebAPI).

**Comment by cgreich** [comment]
I agree. The concept_name records are _not_ case-sensitive. The various vocabulary_id, domain_id, relationship_id and concept_class_id are.

**Comment by fdefalco** [comment]
I think we settled on allowing the database settings to dictate what is returned from a search when we discussed this for WebAPI.  

If the database is setup to be case-sensitive, then the WebAPI currently returns results that are case sensitive.  For example, in our current environment, our database is case-insensitive, so the search returns the same result regardless of the case of the search criteria provided by the user.

The public WebAPI is setup pointing at a PostgreSQL database which is case-sensitive.
